tests/test_source_code_extraction.py

```diff
@@ -111,7 +111,7 @@ def test_source_code_extraction_method(exporter: TestExporter) -> None:
                     'end_time': 2000000000,
                     'attributes': {
                         'code.filepath': 'tests/test_source_code_extraction.py',
-                        'code.lineno': 17,
+                        'code.lineno': 19,
                         'code.function': 'method',
```

Comment:
It looks like this is the only part that differs between the versions, can the test reflect that more clearly instead of these big branches?